### ISSUE TITLE
orthologs and paralogs

### ISSUE THREAD

**Issue by ValWood** [pombase/website#486]
When a gene product has a paralog, 
we need some way of linking the orthologs

so for example
http://pombase2.bioinformatics.nz/gene/SPAC31G5.03 (rps1101)

has orthologs

Homo sapiens | RPS11 (ENSG00000142534) |  
Saccharomyces cerevisiae | RPS11B (YBR048W) |  
Saccharomyces cerevisiae | RPS11A (YDR025W)

but so does rps1102
http://pombase2.bioinformatics.nz/gene/SPAC144.11

On the old site, done like this:
http://www.pombase.org/spombe/result/SPAC144.11#ortholog

**Comment by ValWood** [comment]
I notice these are a small number of pages with "paralog"
http://preview.pombase.org/gene/SPCC330.07c
 
I don't think we displayed this in the old site. There are a small number of "explicit" paralog annotations, when there is no cerevisiea ortholog  but there are Schizo specific duplicates ....

....we could  "infer" genes to display in the paralog section  from above.


I don't think the "curated" paralogs are comprehensive though.

Task1 here:
https://github.com/pombase/curation/issues/810

**Comment by kimrutherford** [comment]
I confused by this.  Could we have a chat about it on Skype sometime?

**Comment by ValWood** [comment]
I hopefully made this clearer with the example from duplicate ticket
https://github.com/pombase/website/issues/1400

removing "discuss' add back if not clear.

**Comment by kimrutherford** [comment]
> I notice these are a small number of pages with "paralog"

Just to be clear, should we keep these manual paralog annotations and infer the others?

I think that paralog section needs work anyway.  For example on this page the table has a duplicate: https://www.pombase.org/gene/SPAC11D3.05

Also that table shows that mfs2/SPAC11D3.05 has two paralogs:
 - https://www.pombase.org/gene/SPBC530.02
 - https://www.pombase.org/gene/SPCC794.04c

But those two genes each have one paralog (mfs2/SPAC11D3.05).  Should SPBC530.02 and SPCC794.04c be annotated as paralogs of each other?  Or doesn't it work like that?

**Comment by kimrutherford** [comment]
I went looking for some examples to understand paralogs better.  Is this the sort of case you were trying to think of?:

  - Human gene SLC17A5 has 11 pombe orthologs: https://www.pombase.org/results/from/id/06044d85-87b9-4007-a989-2841d5b2373c
  - S. cerevisiae gene YGR260W has 4 pombe orthologs: https://www.pombase.org/results/from/id/d90ee89f-2ccd-41c1-9e29-e4bbee7fff1f

The 4 pombe orthologs are a sub set of the 11 human orthologs.

So in this case is there a group of 4 paralogs with respect to cerevisiae and 11 paralogs with respect to human?

**Comment by ValWood** [comment]
exactly, here is another example I found last night:
<img width="1115" alt="Screenshot 2022-04-22 at 05 41 24" src="https://user-images.githubusercontent.com/7359272/164665919-4adefd8c-3ade-4a24-887e-9c4bda4ea326.png">


There are not many examples (fewer than 20 I think) , I think they will all be instances of  when there is duplication after divergence  in  human, and there  was further duplication in the ancestral lineage to pombe and cerevisiae   before divergence (and sometimes after as in this case).

From a pombe/cerevisie perspective atd2 and atd1 are no paralogs, they predate the divergence,  but they are paralogs if we use human as the common ancestor.

**Comment by ValWood** [comment]
I have made a note of all the ones I spotted because I would like to check the ortholog calls for these  at some point soon. Although these might come to light in the Pfam collab.

**Comment by ValWood** [comment]
> > I notice these are a small number of pages with "paralog"
> 
> Just to be clear, should we keep these manual paralog annotations and infer the others?
> 
> I think that paralog section needs work anyway. For example on this page the table has a duplicate: https://www.pombase.org/gene/SPAC11D3.05
> 
> Also that table shows that mfs2/SPAC11D3.05 has two paralogs:
> 
> * https://www.pombase.org/gene/SPBC530.02
> * https://www.pombase.org/gene/SPCC794.04c
> 
> But those two genes each have one paralog (mfs2/SPAC11D3.05). Should SPBC530.02 and SPCC794.04c be annotated as paralogs of each other? Or doesn't it work like that?

These are fixed to:
SPBC530.02
/controlled_curation="term=similar to S. pombe SPCC794.04c and SPAC11D3.05; date=20220422"
	
SPAC11D3.05
/controlled_curation="term=similar to S. pombe SPCC794.04c and SPBC530.02; date=20220422"

SPCC794.04c
/controlled_curation="term=similar to S. pombe SPAC11D3.05 and SPBC530.02; date=20220422"

I also removed the human orthologs, these did not really make sense although they come from some ortholog predictors, this is really a lineage specific family expansion.

Note that we should only require these statements for genes that are duplicated in pombe but are not
annotated as orthologs (because they cannot be inferred)

**Comment by ValWood** [comment]
Because most paralogous groups  are identical  based on  metazoan or fungal divergence, I don't think it would be so bad to select the metazoan/fungal spit as  our reference point. we can state this explicitly in the docs. This would result in atd12 being classed as paralogs, which is true.  For the small number of special cases we could even add a note to say that they have independent orthologs in S. cerevisiae.

**Comment by kimrutherford** [comment]
> There are not many examples (fewer than 20 I think) ,

I did a bit of dodgy Perl and found 98 genes where the group of paralogs for the gene is different if we look at cerevisiae versus human.  The number of groups of paralogs less than half of 98 though so it's not as big as it sounds.

I've attached the output in case it's useful.  There are 45 pairs of lines in the file like this where the paralog groups aren't the same between human and cerevisiae:
```
human:      SPAC12G12.03|SPAC644.16|SPBC16A3.18|SPBC3B9.11c
cerevisiae: SPAC644.16|SPBC3B9.11c

human:      SPAC12G12.03|SPAC644.16|SPBC16A3.18|SPBC3B9.11c
cerevisiae: SPAC12G12.03|SPBC16A3.18
```

Sorry the file doesn't include the human and cerevisiae genes IDs.

File: [paralog-groups.txt](https://github.com/pombase/website/files/8539921/paralog-groups.txt)

**Comment by kimrutherford** [comment]
> I did a bit of dodgy Perl and found 98 genes where the group of paralogs for the gene is different if we look at cerevisiae versus human.

Hi @ValWood.  In pombase/pombase-chado#31 you say:

> We need to use ortholog against S. cerevisiae as the speciation nose to make the call (paralogy differes based on species of comparison)

So does that solve the problem - just use cerevisiae and ignore the human orthologs?

**Comment by ValWood** [comment]
Yes- otherwise it will get confusing. Paralogy vaires wherever you call the point of divergence.
For example if we used bacteria as the spit, nearly every ATPase would become paralogs of each other (i.e any large expanded family). We should document somewhere that our paralogs use the pombe cerevisiae split.

**Comment by kimrutherford** [comment]
> Example
> YJR066W matches both Tor1 and Tor2
> But you don't know that from the Tor1/2 page.

Is this fixed now that the paralogs section shows all the inferred paralogs?

**Comment by ValWood** [comment]
Yes, but it is bizarre that I have annotated like this because we absolutely know which is tor1 and which is tor2  
will move to the curation tracker to fix
